
<launch>
  <node name="camera1_image_publisher" pkg="image_publisher" type='image_publisher_node.py'>
      <param name="publisher_mode" value="file" />
      <param name="video_path" value="/home/ali/Github/Kamal_Dataset/seq5/mono/seq_5_video.h264" />
      <param name="img_transforms" value="to_gray" />
      <param name="fps" value="20" />
      <remap from='/camera/cameraInfo' to='/camera/camera_info' />
      <remap from='/camera/image_raw' to='/camera/image_raw' />
  </node>
  
</launch>   

<!-- /mono_camera/image_raw  : this is the type of image for tagslam -->
<!-- /camera/camera_info  : this is the type of image for tagslam -->
